### DISCUSSION TITLE
How to get the text from the selected item in TreeView?

### DISCUSSION THREAD

**Discussion by hduonggithub** [isl-org/Open3D#4020] [Q&A]
Does anyone know how to get the text from the selected item in TreeView? TreeView just returns the index of selected item only. Thanks!

**Comment by errissa** [comment]
@hduonggithub Are you working in C++ or Python? In C++ you can use the `ItemId` to get the selected widget with `GetItem`. If that widget has a label you can query it.

If you're working in Python it looks like the necessary C++ methods are not exposed and it may not be possible currently to get the widget from its index. I will have to look into this.

**Comment by errissa** [comment]
Created issue to track this.
